### ISSUE TITLE
Recommended landing zone disk size

### ISSUE THREAD

**Issue by emma-makes** [gboudreau/Greyhole#275]
Hello, I've checked the documentation/FAQ but couldn't find a concrete suggestion how much free storage the landing zone should provide.

I'm planning on adding 2*8TB to a pool, which I'll initially fill with about 10TB of data. Can the landing zone be on the same disk as the pool itself and how what's the recommended size of available storage the landing zone should provide?

Thank you very much!

**Comment by gboudreau** [comment]
> Can the landing zone be on the same disk as the pool itself?

Yes. I suggest choosing the fastest drive you have, since the drive speed will affect how fast Greyhole will be able to process new files. If you have a SSD with enough free space, that might be the best choice.

> what's the recommended size of available storage the landing zone should provide?

Depends on your usage.
The Landing Zone should be large enough to accommodate new files that you'll add to your shares, until Greyhole has time to process those, and move them to other drives.

Greyhole normally runs with a lower priority (higher niceness), so it will get less resources to move files away from the LZ, and that means files will accumulate on your LZ drive until Greyhole finishes moving them away.
For example, if you copy 100GB of files onto a Samba share, and Greyhole only had time to move 25GB of those onto other drives, while you were copying the files, that means you'll have at most 75GB of data on your LZ, by the time you finish your copy operation.

I myself use a SSD with 75GB of free space as a LZ.

Of note : to start with existing data, don't use Samba and the LZ; [there are better ways](https://github.com/gboudreau/Greyhole/wiki/ReuseDataDrives). So you don't need to take that initial data into account, when choosing the LZ size.

**Comment by emma-makes** [comment]
That makes sense, thank you very much for the elaborate answer. The drives I'm going to use aren't filled yet, however I can just manually copy the data before adding them to greyhole and I have an SSD with enought storage ready as a landing zone for any future copies!
